fe/fe-core/src/main/java/org/apache/doris/planner/OlapScanNode.java

```diff
@@ -1095,9 +1098,15 @@ protected void toThrift(TPlanNode msg) {
             }
         }
 
+        for (Index index : olapTable.getIndexes()) {
+            TOlapTableIndex tIndex = index.toThrift();
+            indexDesc.add(tIndex);
+        }
+
         msg.node_type = TPlanNodeType.OLAP_SCAN_NODE;
         msg.olap_scan_node = new TOlapScanNode(desc.getId().asInt(), keyColumnNames, keyColumnTypes, isPreAggregation);
         msg.olap_scan_node.setColumnsDesc(columnsDesc);
+        msg.olap_scan_node.setIndexesDesc(indexDesc);
```

Comment:
Adding indexDesc to thrift will add load for query rpc, can we limit it to query with inverted index?